Example/CollectionViewToolsExample/Main/Factories/ContentViewSectionItemsFactory.swift

```diff
@@ -1,133 +1,158 @@
-        factory.cellConfigurationHandler = { cell, cellItem in
-            cell.titleLabel.text = cellItem.object.content.description
-        }
-        factory.sizeConfigurationHandler = { data, collectionView, sectionItem in
-            CGSize(width: collectionView.bounds.width, height: 60)
-        }
-        return factory
-    }()
+       // MARK: - Factories
+
+       // MARK: - ImageContent
+       private(set) lazy var imageCellItemFactory: CellItemFactory = {
+           let factory: AssociatedCellItemFactory<ImageViewState, ImageCollectionViewCell> = makeFactory(id: "image")
+           let cellConfigurationHandler = factory.cellConfigurationHandler
+
+           factory.cellConfigurationHandler = { cell, cellItem in
+               cell.imageView.image = cellItem.object.imageContent.image
+               cell.removeActionHandler = { [weak self] in
+                   self?.removeEventTriggered(state: cellItem.object)
+               }
+               cellConfigurationHandler?(cell, cellItem)
+           }
+
+           factory.sizeConfigurationHandler = { state, collectionView, sectionItem in
+               let width = collectionView.bounds.width
+               let aspectRatio = state.imageContent.image.size.width / state.imageContent.image.size.height
+               return CGSize(width: width, height: width / aspectRatio)
+           }
+           return factory
+       }()
+
+       // MARK: - TextContent
+       private(set) lazy var textCellItemFactory: CellItemFactory = {
+           let factory: AssociatedCellItemFactory<TextViewState, TextCollectionViewCell> = makeFactory(id: "text")
+           let cellConfigurationHandler = factory.cellConfigurationHandler
+
+           factory.cellConfigurationHandler = { cell, cellItem in
+               cell.titleLabel.text = cellItem.object.textContent.text
+               cellConfigurationHandler?(cell, cellItem)
+           }
+
+           factory.sizeConfigurationHandler = { data, collectionView, sectionItem in
+               CGSize(width: collectionView.bounds.width, height: 60)
+           }
+           return factory
+       }()
+
+       // MARK: - Divider
+       private(set) lazy var dividerCellItemFactory: CellItemFactory = {
+           let factory: AssociatedCellItemFactory<DividerState, DividerCell> = .init()
+
+           factory.cellConfigurationHandler = { cell, _ in
+               cell.dividerHeight = 1
+               cell.dividerView.backgroundColor = .gray
+           }
+
+           factory.sizeConfigurationHandler = {_, collectionView, sectionItem in
+               .init(width: collectionView.bounds.inset(by: sectionItem.insets).width, height: 1)
+           }
+           return factory
+       }()
+
+       // MARK: - Content
+       private(set) lazy var cellItemFactory: CellItemFactory = {
+           imageCellItemFactory.factory(byJoining: textCellItemFactory)
+                               .factory(byJoining: dividerCellItemFactory)
+       }()
```

Comment:
Let's replace all 1-line `init`s with this?
```suggestion
       private(set) lazy var cellItemFactory: CellItemFactory = imageCellItemFactory.factory(byJoining: textCellItemFactory).factory(byJoining: dividerCellItemFactory)
```